PropertyChanged/PropertyChanged.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard1.0</TargetFrameworks>
+    <TargetFrameworks>net40;net472;netstandard1.0;netstandard2.0</TargetFrameworks>
```

Comment:
I don't think `net472` is necessary... Is there a reason to add it?

Reply:
fixed